tests/easier68k/core/opcodes/test_sub.py

```diff
@@ -32,18 +33,20 @@ def test_sub():
 
     sub = Sub(params, OpSize.WORD)  # SUB.W #52,D0
 
-    sub.execute(sim)
+    run_opcode_test(sim, sub, Register.D0, 71, 4, [0, 0, 0, 0, 0])
```

Comment:
these are ints, yet previously stated that this method accepted a list of bools. I think bools are more appropriate

Reply:
I agree, I just didn't want to have to type out "False" for each entry since "0" was just quicker. Will update all function calls to use either "True" or "False" instead of "1" or "0".